src/fungible_token.rs

```diff
@@ -420,8 +420,24 @@ impl FungibleToken {
         }
     }
 
+    /// Insert account.
+    /// Calculate total unique accounts
     pub fn accounts_insert(&self, account_id: &str, amount: Balance) {
-        sdk::save_contract(&Self::account_to_key(account_id), &amount)
+        if !self.accounts_contains_key(account_id) {
+            let key = Self::get_statistic_key();
```

Comment:
Let's use either the subkey or an explicit name for the accounts counter (See below for details).